Plan/common/src/main/resources/assets/plan/web/css/sb-admin-2.css

```diff
@@ -1,9640 +1,1926 @@
-    color: #45464e;
-    background-color: #cfcfd6;
-}
-
-.list-group-item-secondary.list-group-item-action.active {
-    color: #fff;
-    background-color: #45464e;
-    border-color: #45464e;
-}
-
-.list-group-item-success {
-    color: #0f6848;
-    background-color: #bff0de;
-}
-
-.list-group-item-success.list-group-item-action:hover, .list-group-item-success.list-group-item-action:focus {
-    color: #0f6848;
-    background-color: #aaebd3;
-}
-
-.list-group-item-success.list-group-item-action.active {
-    color: #fff;
-    background-color: #0f6848;
-    border-color: #0f6848;
-}
-
-.list-group-item-info {
-    color: #1c606a;
-    background-color: #c7ebf1;
-}
-
-.list-group-item-info.list-group-item-action:hover, .list-group-item-info.list-group-item-action:focus {
-    color: #1c606a;
-    background-color: #b3e4ec;
-}
-
-.list-group-item-info.list-group-item-action.active {
-    color: #fff;
-    background-color: #1c606a;
-    border-color: #1c606a;
-}
-
-.list-group-item-warning {
-    color: #806520;
-    background-color: #fceec9;
-}
-
-.list-group-item-warning.list-group-item-action:hover, .list-group-item-warning.list-group-item-action:focus {
-    color: #806520;
-    background-color: #fbe6b1;
-}
-
-.list-group-item-warning.list-group-item-action.active {
-    color: #fff;
-    background-color: #806520;
-    border-color: #806520;
-}
-
-.list-group-item-danger {
-    color: #78261f;
-    background-color: #f8ccc8;
-}
-
-.list-group-item-danger.list-group-item-action:hover, .list-group-item-danger.list-group-item-action:focus {
-    color: #78261f;
-    background-color: #f5b7b1;
-}
-
-.list-group-item-danger.list-group-item-action.active {
-    color: #fff;
-    background-color: #78261f;
-    border-color: #78261f;
-}
-
-.list-group-item-light {
-    color: #818183;
-    background-color: #fdfdfe;
-}
-
-.list-group-item-light.list-group-item-action:hover, .list-group-item-light.list-group-item-action:focus {
-    color: #818183;
-    background-color: #ececf6;
-}
-
-.list-group-item-light.list-group-item-action.active {
-    color: #fff;
-    background-color: #818183;
-    border-color: #818183;
-}
-
-.list-group-item-dark {
-    color: #2f3037;
-    background-color: #d1d1d5;
-}
-
-.list-group-item-dark.list-group-item-action:hover, .list-group-item-dark.list-group-item-action:focus {
-    color: #2f3037;
-    background-color: #c4c4c9;
-}
-
-.list-group-item-dark.list-group-item-action.active {
-    color: #fff;
-    background-color: #2f3037;
-    border-color: #2f3037;
-}
-
-.close {
-    float: right;
-    font-size: 1.5rem;
-    font-weight: 700;
-    line-height: 1;
-    color: #000;
-    text-shadow: 0 1px 0 #fff;
-    opacity: .5;
-}
-
-.close:hover {
-    color: #000;
-    text-decoration: none;
-}
-
-.close:not(:disabled):not(.disabled):hover, .close:not(:disabled):not(.disabled):focus {
-    opacity: .75;
-}
-
-button.close {
```

Comment:
Removal causes version modal close button to appear as a button
![image](https://user-images.githubusercontent.com/24460436/114141761-b2a09580-991a-11eb-91dc-066a39c7cfd7.png)

I'm guessing the button was not changed in VersionChecker.java to have .btn-close similar to the other modals